src/Footer.js

```diff
@@ -0,0 +1,3 @@
+import {Component} from 'react';
```

Comment:
Hey there, you are on your way with this!

On my end, I am not able to get the app and running due to errors in the file.

I would check that your import statement says:

`import React, {Component} from 'react'

Reply:
I don't think the errors are stemming from this particular import of Component 👍